plexpy/datafactory.py

```diff
@@ -1225,6 +1242,19 @@ def get_user_stats(self, rating_key=None, grouping=None):
         
         group_by = 'session_history.reference_id' if grouping else 'session_history.id'
 
+        _rating_keys = []
+        if media_type == 'collection':
+            pms_connect = pmsconnect.PmsConnect()
+            result = pms_connect.get_item_children(rating_key=rating_key)
+
+            for child in result['children_list']:
+                _rating_keys.append(child['rating_key'])
+        else:
+            _rating_keys.append(rating_key)
+
+        rating_keys = ','.join(_rating_keys)
+        rating_keys_arg = ','.join(['?'] * len(rating_keys))
```

Comment:
Forgot the underscore.
```suggestion
        rating_keys_arg = ','.join(['?'] * len(_rating_keys))
```

Reply:
This will still result in an error: `Incorrect number of bindings supplied. The current statement uses 39, and there are 3 supplied..` Do to generating e.g. 13 times `?` for rating_keys_arg replaced in the statement 3 times = 39 `?` and only providing 3 arguments in the select call.

Reply:
This should work
```python
        rating_keys = []
        if media_type == 'collection':
            pms_connect = pmsconnect.PmsConnect()
            result = pms_connect.get_item_children(rating_key=rating_key)

            for child in result['children_list']:
                rating_keys.append(child['rating_key'])
        else:
            rating_keys.append(rating_key)
            
        rating_keys_arg = ','.join(['?'] * len(rating_keys))
```
Then
```python
                result = monitor_db.select(query, args=rating_keys * 3)
```